unity5.3.5f1でビルド出来ない

## 不具合の概要(Description of bug)

- SDKのバージョン(SDK version)
 - 2.1.1
- 不具合が発生した環境(Environment requirement)
 - unity5.3.5f1
- 不具合を再現させる手順・サンプルコード(Step for Reproduce bug / Sample code)
 - クイックスタートを unity5.3.5f1で実施
- エラーメッセージ(Error messages)
```
Assets/NCMB/NCMBManager.cs(230,87): error CS1502: The best overloaded method match for System.IO.FileNotFoundException.FileNotFoundException(string, string)' has some invalid arguments Assets/NCMB/NCMBManager.cs(230,87): error CS1503: Argument#2' cannot convert System.IO.FileNotFoundException' expression to typestring'
```

--- a/ncmb_unity/Assets/NCMB/NCMBManager.cs
+++ b/ncmb_unity/Assets/NCMB/NCMBManager.cs
@@ -227,7 +227,7 @@ namespace NCMB
 				}
 				return path;
 			} catch (FileNotFoundException e) {
-				throw new FileNotFoundException ("File not found.", e);
+				throw e;
 			}
 		}
 
@@ -327,15 +327,13 @@ namespace NCMB
 				} catch (Exception tryReadError) {
 					if (tryReadError != null) {
 						path = NCMBSettings.currentInstallationPath;//Unityからのアクセス権があり、環境に依存しないパスを設定
-					try {
-						StreamReader sr = new StreamReader (
-							                  path, Encoding.GetEncoding ("UTF-8"));	
-						text = sr.ReadToEnd ();
-						sr.Close ();
-
-					} catch (FileNotFoundException readError) {
-						throw new FileNotFoundException ("File read error.", readError);
-					}
+						try {
+							StreamReader sr = new StreamReader (path, Encoding.GetEncoding ("UTF-8"));	
+							text = sr.ReadToEnd ();
+							sr.Close ();
+						} catch (FileNotFoundException readError) {
+							throw readError;
+						}
 					}
 				}
 			}

